wpeverest/everest-forms
# Fix - Booked Slot not deleting after entry deletion.

### All Submissions:


### Changes proposed in this Pull Request:

Previously, if we booked a slot and that booked slot was allocated even if we deleted that booked slot entry. This PR fixes this issue.

### How to test the changes in this Pull Request:

1. Create a form-> drag date field->enable slot booking
2. Fill all fields->submit form-> go to entry-> delete that submitted entry (permanent delete)
3. Go to form and try to book the same date and verify.

### Types of changes:


### Other information:


### Changelog entry

> Fix - Booked slot not deleted after deletion of booked slot entry.
